### PR TITLE
Fix resource generation to work with dotnet test

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @dougbu

File: test/Microsoft.AspNetCore.Mvc.TestCommon/ResourceFile.cs [hunk 1]
```diff
@@ -190,10 +188,7 @@ public static void UpdateFile(Assembly assembly, string resourceName, string pre
 
             if (!string.Equals(normalizedPreviousContent, normalizedContent, StringComparison.Ordinal))
             {
-                // The DNX runtime compiles every file under the resources folder as a resource available at runtime
-                // with the same name as the file name. Need to update this file on disc.
```
Comment (dougbu) [inline test/Microsoft.AspNetCore.Mvc.TestCommon/ResourceFile.cs]: We still need _some_ comment explaining what's going on here.

**Comment by dougbu** [comment]
:shipit:

File: test/Microsoft.AspNetCore.Mvc.TestCommon/ResourceFile.cs (hunk 1 above)
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.TestCommon/ResourceFile.cs]: > // The build system compiles every file under the resources folder as a resource available at runtime
> // with the same name as the file name. Need to update this file on disc.

Sound good?

**Pull request closed without merging**